docs/DeveloperGuide.adoc

```diff
@@ -721,17 +721,38 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
-
-|`* * *` |user |add a new person |
-
-|`* * *` |user |delete a person |remove entries that I no longer need
-
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
-
-|`* *` |user |hide link:#private-contact-detail[private contact details] by default |minimize chance of someone else seeing them by accident
-
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
+|`* * *` |user |see my contacts while managing my address book |remember what I need to do without flipping back and forth
+|`* * *` |user |backup the contacts into a xml file |transfer my contact details with another user or load the existing details to another computer of mine
+|`* * *` |user |create new contacts in the address book |add more new contacts in the future
+|`* * *` |user |type commands that do not follow the format |use them without memorizing the formats
+|`* * *` |user |merge duplicate contacts |use my addressbook without clutter
+|`* * *` |user |group my contacts in more than one grouping |search my contacts easily by groups
+|`* * *` |user |filter and search contacts |display important contacts
+|`* * *` |user |update existing contacts from the address book |update existing contacts if they have changed their contact number or details
+|`* * *` |user |add a short description  |recall details about them
+|`* * *` |user |view all my contacts |organise my contacts easily
+|`* * *` |user |group my users |organize my address book
```

Comment:
Sounds similar to `* * * |user |group my contacts in more than one grouping |search my contacts easily by groups` ?